### PR TITLE
Migration back from woo.com to woocommerce.com in URLs

### LINKED ISSUE #12430: Migration back to woocommerce.com
**Description**
It has been decided to migrate back from woo.com to woocommerce.com. For this reason, we should update the URLs in the app to point to the new domain, rather than go through the redirect.

More info here: peaMlT-w5
Old PR: https://github.com/woocommerce/woocommerce-ios/pull/11182

### PR DESCRIPTION
**Pull request by pmusolino**
Closes: #12430

## Description
It has been decided to migrate back from woo.com to woocommerce.com. For this reason, we should update the URLs and some contents in the app to point to the new domain, rather than go through the redirect.

More info on this decision here: peaMlT-w5

## Testing instructions
These changes are widespread, and it doesn't seem practical to verify every link within the app. As I've reviewed them, I've opened each one in a web browser to ensure they all lead to the correct destinations. Currently, a redirect is active. I would suggest spot-checking a selection of the changes and also testing some within the app.

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-ios/milestone/194">18.2</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift [hunk 1]
```diff
@@ -135,6 +135,6 @@ private extension JetpackErrorViewModel {
     }
 
     enum Strings {
-        static let instructionsURLString = "https://woo.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
+        static let instructionsURLString = "https://woocommerce.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
```
Comment (itsmeichigo) [inline WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift]: This link is not used anywhere, we should consider removing it.

File: WooCommerce/Classes/Authentication/Navigation Exceptions/WrongAccountErrorViewModel.swift [hunk 2]
```diff
@@ -310,7 +310,7 @@ private extension WrongAccountErrorViewModel {
     }
 
     enum Strings {
-        static let instructionsURLString = "https://woo.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
+        static let instructionsURLString = "https://woocommerce.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
```
Comment (itsmeichigo) [inline WooCommerce/Classes/Authentication/Navigation Exceptions/WrongAccountErrorViewModel.swift]: This link is also not used.

File: WooCommerce/Classes/ViewRelated/Fancy Alerts/FancyAlertViewController+UnifiedLogin.swift [hunk 3]
```diff
@@ -145,7 +145,7 @@ private extension FancyAlertViewController {
     }
 
     enum Strings {
-        static let instructionsURLString = "https://woo.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
+        static let instructionsURLString = "https://woocommerce.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/"
```
Comment (itsmeichigo) [inline WooCommerce/Classes/ViewRelated/Fancy Alerts/FancyAlertViewController+UnifiedLogin.swift]: This link is no longer used, we should remove it.

**Review by itsmeichigo — APPROVED** [review]
LGTM :shipit:  I also added some comments about unused URLs which should be removed below.

**Comment by pmusolino** [comment]
Thank you @itsmeichigo for the review! I removed the unused URLs a38a5daa7a4aedb21598aedc57c1d0c1c6e0535e

**Comment by itsmeichigo** [comment]
Thanks for the update - please feel free to merge after resolving the merge conflict!

**Pull request merged**